addons/mail/data/mail_demo.xml

```diff
@@ -22,7 +17,7 @@
     Do you have a simple solution with servers running on Redhat Linux?
 </p>
 <p>Best regards,</p>
-<pre>-- 
+<pre>--
```

Comment:
According to a Princetown study, all signatures use "--<space><line break><Some guy name>". You just destroyed that study. Are you a Trump's puppet trying to destroy all research in the world ?